skins/laika/src/store/membership_fee/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Module } from 'vuex';
+import { Payment } from '@/view_models/Payment';
+import { Validity } from '@/view_models/Validity';
+import { actions } from '@/store/payment/actions';
+import { getters } from '@/store/payment/getters';
+import { mutations } from '@/store/payment/mutations';
+
+export default function (): Module<Payment, any> {
+	const state: Payment = {
+		validity: {
+			fee: Validity.INCOMPLETE,
+			interval: Validity.INCOMPLETE,
+			type: Validity.VALID,
+		},
+		values: {
+			fee: '', // membership fee in cents
```

Comment:
how about we name it `feeInCents`?

Reply:
Thanks for pointing it out, I also like to use code instead of comments. I won't update it now since I'll be working on the banner, but feel free to change it in subsequent PRs.